aviary/interface/plot_drag_polar.py

```diff
@@ -0,0 +1,206 @@
+
+    toolbar = NavigationToolbar2Tk(canvas, window)
+
+    def plot_polar(ax, CD, CL, color, label=None, marker='o'):
+        ax.plot(CD, CL, color=color, label=label, marker='o')
+
+    def update_plot():
+        x_var = set_x_var.get()
+        y_var = set_y_var.get()
+        fix_variable = fix_variable_var.get()
+        fix_value = float(fix_value_var.get())
+        ax.clear()
+
+        if fix_variable == 'Mach':
+
+            indices = mach == fix_value
+
+            fixed_values = altitude_values
+            fixed_label = 'Altitude'
+        else:
+
+            index = altitude == fix_value
+            fixed_values = mach_values
+            fixed_label = 'Mach'
+
+        colors = cm.viridis(np.linspace(0, 1, len(fixed_values)))
+        for i, val in enumerate(fixed_values):
+            if fix_variable == 'Mach':
+                indices = (mach == fix_value) & (altitude == val)
+
+                CD = np.array(CD_values[indices])
+
+                CL = CL_values[indices]
+
+                alpha = alpha_values[indices]
+
+            else:
+                index = (altitude == fix_value) & (mach == val)
+
+                CD = np.array(CD_values[index])
+
+                CL = CL_values[index]
+
+                alpha = alpha_values[index]
+
+            if x_var == 'CD':
+                x_val = CD
+
+            elif x_var == 'CL':
+                x_val = CL
+
+            elif x_var == 'Alpha':
+                x_val = alpha
+
+            elif x_var == 'CL/CD':
+                x_val = np.array(CL)/np.array(CD)
+
+            if y_var == 'CD':
+                y_val = CD
+
+            elif y_var == 'CL':
+                y_val = CL
+
+            elif y_var == 'Alpha':
+                y_val = alpha
+
+            elif y_var == 'CL/CD':
+                y_val = np.array(CL)/np.array(CD)
+
+            plot_polar(ax, x_val, y_val, color=colors[i], label=f'{fixed_label} {val}')
+
+            ax.set_xlabel(f'{x_var}')
+            ax.set_ylabel(f'{y_var}')
+            ax.legend(loc='center left', bbox_to_anchor=(1, 0.5), title=fixed_label)
+            ax.set_title(f'{y_var} vs {x_var} for fixed {fix_variable} = {fix_value}')
+
+            canvas.draw()
+            toolbar.update()
+
+    def update_fix_value_combobox(*args):
+        if fix_variable_var.get() == 'Mach':
+            fix_value_combobox['values'] = [str(value) for value in mach_values]
+            fix_value_var.set(str(mach_values[0]))
+            fix_value_combobox.current(0)
+        else:
+            fix_value_combobox['values'] = [str(value) for value in altitude_values]
+            fix_value_var.set(str(altitude_values[0]))
+
+    set_x_var = StringVar(value='CD')
+
+    set_x_label = Label(master=window, text="x-axis")
+    set_x_label.pack()
+    set_x_combobox = Combobox(master=window, textvariable=set_x_var, values=[
+        'CD', 'CL', 'Alpha', 'CL/CD'])
+
+    set_x_combobox.pack()

```

Comment:
could you use `side = LEFT` for all of these so that these all fit on one line instead of stacking vertically?
This will give more space for the plot

Reply:
Fixed and added spacing between comboboxes